ember-addon-main.js

```diff
@@ -3,6 +3,10 @@ require('validate-peer-dependencies')(__dirname);
 let VersionChecker = require('ember-cli-version-checker');
 let { addPlugin, hasPlugin } = require('ember-cli-babel-plugin-helpers');
 
+// eslint-disable-next-line node/no-unpublished-require
+const emberAddon = require('ember-source');
+const templateCompiler = emberAddon.absolutePaths.templateCompiler;
```

Comment:
Eeeck, I'm sorry for the confusion. I was saying your original approach was great, and it represented public API from our perspective (not that you should swap to `require('ember-source')`). Very sorry for the confusion there.